### PR TITLE
Make in bounty green color (bounty_score)

### PR DESCRIPTION
**Pull request by RyanZem**
bounty

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

**Comment by xenonca** [comment]
I would only make the numbers green IMO and `points!` yellow again.

**Comment by JostP** [comment]
I would suggest different color or at least different green color. I don't really like color on screenshot, it does not really suit to blue, red and yellow color.

**Comment by RyanZem** [comment]
> I would only make the numbers green IMO and `points!` yellow again.

No, I wanted green score and "points"

**Comment by RyanZem** [comment]
> I would suggest different color or at least different green color. I don't really like color on screenshot, it does not really suit to blue, red and yellow color.

OK, can you tell me what color do you want?

**Comment by xenonca** [comment]
> I would suggest different color or at least different green color. I don't really like color on screenshot, it does not really suit to blue, red and yellow color.

I think it's meant to stand out as it is a rather important  information in the bounty message. Also I guess that it's green because it's the same color that the score HUD elements have that you see when getting points

**Comment by RyanZem** [comment]
bounty

**Comment by RyanZem** [comment]
a
b

**Comment by RyanZem** [comment]
@JostP @xenonca I changed color and made "points!" yellow :)

File: mods/ctf/ctf_bounties/init.lua [hunk 1]
```diff
@@ -129,7 +131,9 @@ ctf.register_on_killedplayer(function(victim, killer)
 	local msg = minetest.colorize(killer_color, killer) ..
 		minetest.colorize("#fff326", " has killed ") ..
 		minetest.colorize(victim_color, victim) ..
-		minetest.colorize("#fff326", " and received " .. bounty_score .. " points!")
+		minetest.colorize("#fff326", " and received " ) ..
+                minetest.colorize("#7efc00", bounty_score) ..
+                minetest.colorize("#fff326", " points!")
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_bounties/init.lua:136]: Why is this indented so much?

Comment (LoneWolfHT) [inline mods/ctf/ctf_bounties/init.lua:136]:
```suggestion
		minetest.colorize("#fff326", " and received " ) ..
		minetest.colorize("#7efc00", bounty_score) ..
		minetest.colorize("#fff326", " points!")
```

**Review by Panquesito7 — APPROVED** [review]

**Review by Toby1710 — APPROVED** [review]

**Pull request merged**